### PR TITLE
Update file rotation criteria to be in seconds, change names

### PR DESCRIPTION
**Pull request by aasted**

### CODE REVIEW

File: writer/writer_controller.go [hunk 1]
```diff
@@ -16,11 +16,11 @@ const (
 )
 
 var (
-	MaxLogSize     = loadFromEnv("MAX_LOG_SIZE", 1<<32)                                     // default 4GB
-	MaxTimeAllowed = time.Duration(loadFromEnv("MAX_LOG_LIFETIME_MINS", 300)) * time.Minute // default 5 hours
+	maxLogSize        = loadFromEnv("MAX_LOG_BYTES", 1<<32)                                  // default 4GB
```
Comment (crxpandion) [inline writer/writer_controller.go]: similar to the other comment in the spade_edge CR this should be "loadIntFromEnv"

**Comment by crxpandion** [comment]
just that comment - besides this is looks good.

**Comment by crxpandion** [comment]
do you want validation so weird stuff doesnt happen by accident? Like rotating every nanosecond for example?

**Comment by aasted** [comment]
I think we punt on the validation for the moment; people might have odd use cases that require what look like pathological configurations to us.

Warnings might be good for values that are truly strange.

**Pull request merged**